public/index.html

```diff
@@ -161,7 +161,6 @@
           -ms-user-select: auto;
           user-select: auto;
         }
-      }
```

Comment:
what's this?

Reply:
Was a typo in prod due to which the media rules were never getting applied, fixed now